tiatoolbox/cli/common.py

```diff
@@ -32,14 +32,18 @@ def cli_img_input(
     multiple: bool = False,
 ) -> callable:
     """Enables --img-input option for cli."""
+    if multiple:
+        usage_help = usage_help + " Multiple instances may be provided."
     return click.option("--img-input", help=usage_help, type=str, multiple=multiple)
 
 
 def cli_name(
-    usage_help: str = "user defined name to be used as an identifier",
+    usage_help: str = "User defined name to be used as an identifier.",
     multiple: bool = False,
 ) -> callable:
     """enables --name option for cli"""
+    if multiple:
```

Comment:
This line is not fully covered in coverage. Please add a test for False condition to cover this line.

Reply:
I do not know how to test a click.option in isolation at the moment. If I figure it out I will add it.

Reply:
Now done